includes/sanitizers/class-amp-img-sanitizer.php

```diff
@@ -203,6 +203,12 @@ private function filter_attributes( $attributes ) {
 					$out['noloading'] = $value;
 					break;
 
+				// Skip directly copying new web platform attributes from img to amp-img which are largely handled by AMP already.
+				case 'importance': // Not supported by AMP.
+				case 'loading': // Lazy-loading handled by amp-img natively.
+				case 'intrinsicsize': // Responsive images handled by amp-img directly.
```

Comment:
Left-over from reverted `intrinsicsize` support here.

Reply:
Ah right, that makes sense. I failed to read the context of this, I was just triggered by the word. ;)